docs/src/main/asciidoc/stork-reference.adoc

```diff
@@ -72,3 +72,70 @@ To learn about custom service discovery and service selection, check:
 
 - https://smallrye.io/smallrye-stork/latest/service-discovery/custom-service-discovery/[Implement a custom service discover provider]
 - https://smallrye.io/smallrye-stork/latest/load-balancer/custom-load-balancer/[Implement a custom service selection provider]
+
+== Configure Stork observability
```

Comment:
Add a entry to the micrometer guide, please. At the bottom there's a list of supported libs.

Reply:
Done in the last commit :-)